gui/window.go

```diff
@@ -742,6 +742,24 @@ func (window *Window) launchMenuView() {
 	}
 }
 
+// writeToConfirmInstallDialog is a helper function to write to dialog for confirm installation window
+func writeToConfirmInstallDialog(buffer *gtk.TextBuffer, dryRunResults *storage.DryRunType) {
+	for _, media := range *dryRunResults.OtherMediaResults {
+		log.Debug("OtherMediaChange: %s", media)
+		buffer.Insert(buffer.GetEndIter(), media+"\n")
+	}
+	if len(*dryRunResults.OtherMediaResults) == 0 {
+		buffer.Insert(buffer.GetEndIter(), "No side effects on other disks"+"\n\n")
```

Comment:
These user displayed strings should not be created in the graphical UI as then they need to be repeated in the text-based UI. Also, all graphical UI strings need to be translated.

Reply:
Done..moved then back to storage code.